app/src/main/java/org/dhis2/bindings/ValidationStrategyExtensions.kt

```diff
@@ -1,9 +1,18 @@
 package org.dhis2.bindings
 
+import org.dhis2.form.model.EventMode
 import org.hisp.dhis.android.core.common.ValidationStrategy
 
-fun ValidationStrategy.canSkipErrorFix(hasErrorFields: Boolean, hasEmptyMandatoryFields: Boolean) =
+fun ValidationStrategy.canSkipErrorFix(
```

Comment:
I think that this extensions it's only used in `EventCapturePresenterImpl` why don't we refactor it as a private fun into the presenter?